extensions/sql_table/js/sql_table.html

```diff
@@ -0,0 +1,246 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/sql.js/1.8.0/sql-wasm.js"></script>
```

Comment:
any options to prevent multiple loading of this file for pages with multiple tables? 
16KB is not a thing, but imagine a page with a dozen tables?

Reply:
this file will be added once to the end of the page. not everytime a table is rendered.  so it'll be loaded only once